diff --git a/node_modules/blinkinput-react-native/parsers/regexParser.js b/node_modules/blinkinput-react-native/parsers/regexParser.js
index af88a89..60b1e61 100644
--- a/node_modules/blinkinput-react-native/parsers/regexParser.js
+++ b/node_modules/blinkinput-react-native/parsers/regexParser.js
@@ -1,6 +1,7 @@
 import { Parser, ParserResult } from '../parser'
 import {
-    Date
+    Date,
+    BaseOcrEngineOptions,
 } from '../types'
 
 /**
diff --git a/node_modules/blinkinput-react-native/src/android/.project b/node_modules/blinkinput-react-native/src/android/.project
new file mode 100644
index 0000000..cf7ff1a
--- /dev/null
+++ b/node_modules/blinkinput-react-native/src/android/.project
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>blinkinput-react-native</name>
+	<comment>Project blinkinput-react-native created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+	<filteredResources>
+		<filter>
+			<id>1712140299632</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>
diff --git a/node_modules/blinkinput-react-native/src/android/bin/.project b/node_modules/blinkinput-react-native/src/android/bin/.project
new file mode 100644
index 0000000..cf7ff1a
--- /dev/null
+++ b/node_modules/blinkinput-react-native/src/android/bin/.project
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>blinkinput-react-native</name>
+	<comment>Project blinkinput-react-native created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+	<filteredResources>
+		<filter>
+			<id>1712140299632</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>
diff --git a/node_modules/blinkinput-react-native/src/android/bin/build.gradle b/node_modules/blinkinput-react-native/src/android/bin/build.gradle
new file mode 100644
index 0000000..fe09343
--- /dev/null
+++ b/node_modules/blinkinput-react-native/src/android/bin/build.gradle
@@ -0,0 +1,25 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 28
+    buildToolsVersion "28.0.3"
+
+    defaultConfig {
+        minSdkVersion 16
+        targetSdkVersion 28
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation 'com.facebook.react:react-native:+'
+    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0-alpha02'
+    implementation('com.microblink:blinkinput:4.3.0@aar') {
+        transitive = true
+    }
+}
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/AndroidManifest.xml b/node_modules/blinkinput-react-native/src/android/bin/src/main/AndroidManifest.xml
new file mode 100644
index 0000000..fbe7b95
--- /dev/null
+++ b/node_modules/blinkinput-react-native/src/android/bin/src/main/AndroidManifest.xml
@@ -0,0 +1,16 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.microblink.reactnative"
+    android:versionCode="1"
+    android:versionName="1.0">
+
+    <uses-permission android:name="android.permission.CAMERA"/>
+    <uses-permission android:name="android.permission.INTERNET"/>
+
+    <uses-feature
+        android:name="android.hardware.camera"
+        android:required="false"/>
+    <uses-feature
+        android:name="android.hardware.camera.autofocus"
+        android:required="false"/>
+
+</manifest>
\ No newline at end of file
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/MicroblinkModule.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/MicroblinkModule.class
new file mode 100644
index 0000000..7713a31
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/MicroblinkModule.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/MicroblinkReactPackage.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/MicroblinkReactPackage.class
new file mode 100644
index 0000000..2ab41e4
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/MicroblinkReactPackage.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/SerializationUtils.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/SerializationUtils.class
new file mode 100644
index 0000000..a5c15cd
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/SerializationUtils.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/OverlaySettingsSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/OverlaySettingsSerialization.class
new file mode 100644
index 0000000..0963f10
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/OverlaySettingsSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/OverlaySettingsSerializers.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/OverlaySettingsSerializers.class
new file mode 100644
index 0000000..d206177
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/OverlaySettingsSerializers.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/serialization/BarcodeOverlaySettingsSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/serialization/BarcodeOverlaySettingsSerialization.class
new file mode 100644
index 0000000..b2969fe
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/serialization/BarcodeOverlaySettingsSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/serialization/OverlaySerializationUtils.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/serialization/OverlaySerializationUtils.class
new file mode 100644
index 0000000..32f88a0
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/overlays/serialization/OverlaySerializationUtils.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/ParserSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/ParserSerialization.class
new file mode 100644
index 0000000..2e361b4
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/ParserSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/ParserSerializers.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/ParserSerializers.class
new file mode 100644
index 0000000..f41bb73
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/ParserSerializers.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/AmountParserSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/AmountParserSerialization.class
new file mode 100644
index 0000000..269a452
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/AmountParserSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/DateParserSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/DateParserSerialization.class
new file mode 100644
index 0000000..93239ba
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/DateParserSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/EmailParserSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/EmailParserSerialization.class
new file mode 100644
index 0000000..5c086b6
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/EmailParserSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/IbanParserSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/IbanParserSerialization.class
new file mode 100644
index 0000000..578692e
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/IbanParserSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/LicensePlatesParserSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/LicensePlatesParserSerialization.class
new file mode 100644
index 0000000..7dafcdb
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/LicensePlatesParserSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/RawParserSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/RawParserSerialization.class
new file mode 100644
index 0000000..ccc0deb
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/RawParserSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/RegexParserSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/RegexParserSerialization.class
new file mode 100644
index 0000000..c0ae095
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/RegexParserSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/TopUpParserSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/TopUpParserSerialization.class
new file mode 100644
index 0000000..d1675fe
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/TopUpParserSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/VinParserSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/VinParserSerialization.class
new file mode 100644
index 0000000..ef6d70a
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/parsers/serialization/VinParserSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/RecognizerSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/RecognizerSerialization.class
new file mode 100644
index 0000000..d2007d5
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/RecognizerSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/RecognizerSerializers.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/RecognizerSerializers.class
new file mode 100644
index 0000000..b6d53ef
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/RecognizerSerializers.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/BarcodeRecognizerSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/BarcodeRecognizerSerialization.class
new file mode 100644
index 0000000..d526782
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/BarcodeRecognizerSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/DocumentCaptureRecognizerSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/DocumentCaptureRecognizerSerialization.class
new file mode 100644
index 0000000..7956348
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/DocumentCaptureRecognizerSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/Pdf417RecognizerSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/Pdf417RecognizerSerialization.class
new file mode 100644
index 0000000..6c98b7c
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/Pdf417RecognizerSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/SimNumberRecognizerSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/SimNumberRecognizerSerialization.class
new file mode 100644
index 0000000..f118805
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/SimNumberRecognizerSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/SuccessFrameGrabberRecognizerSerialization.class b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/SuccessFrameGrabberRecognizerSerialization.class
new file mode 100644
index 0000000..55e940b
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/java/com/microblink/reactnative/recognizers/serialization/SuccessFrameGrabberRecognizerSerialization.class differ
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/res/layout/mb_layout_camera_splash.xml b/node_modules/blinkinput-react-native/src/android/bin/src/main/res/layout/mb_layout_camera_splash.xml
new file mode 100644
index 0000000..2b00cea
--- /dev/null
+++ b/node_modules/blinkinput-react-native/src/android/bin/src/main/res/layout/mb_layout_camera_splash.xml
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+             xmlns:tools="http://schemas.android.com/tools"
+             android:layout_width="match_parent"
+             android:layout_height="match_parent"
+             android:background="#000000"
+             tools:ignore="Overdraw">
+</FrameLayout>
\ No newline at end of file
diff --git a/node_modules/blinkinput-react-native/src/android/bin/src/main/res/raw/beep.mp3 b/node_modules/blinkinput-react-native/src/android/bin/src/main/res/raw/beep.mp3
new file mode 100644
index 0000000..60bbca0
Binary files /dev/null and b/node_modules/blinkinput-react-native/src/android/bin/src/main/res/raw/beep.mp3 differ
